### ISSUE TITLE
Settings to reproduce results from the paper

### ISSUE THREAD

**Issue by bdhingra** [facebookresearch/poincare-embeddings#3]
Hi,

Thanks for releasing the code. 

Can you also provide the hyper-parameter settings for reproducing results in Table 1 in the paper (specifically -lr, -negs, -batchsize, -burnin)? The example script is only for the mammals subset. Are these the same across all the experiments described in the paper?

Thanks,
Bhuwan

**Comment by mnick** [comment]
Hyperparameters to reproduce the mean rank results for 10 dimensional embeddings are now included in 9aecb8634abbbc23338c2b5e5bb61ca517581bef (in `train-nouns.sh`)

These settings should produce embeddings with mean rank of approx 4.

The hyperparameters for the MAP results are included as a comment in the `train-nouns.sh` script (the only difference is a larger learning rate). These settings should produce embeddings with a MAP of 0.87, which is higher than the number reported in the paper.
